spec/graphql/language/lexer_spec.rb

```diff
@@ -26,12 +26,18 @@
       assert_equal tokens[0], tokens[1].prev_token
     end
 
+    it "allows escaped quotes in strings" do
+      tokens = subject.tokenize('"a\\"b""c"')
+      assert_equal 'a"b', tokens[0].value
+      assert_equal 'c', tokens[1].value
+    end
```

Comment:
This test isn't related to the PR, but coverage was missing and I wrote it as part of figuring out what to do.